lms/templates/discussion/_user_profile.html

```diff
@@ -3,7 +3,8 @@
 <div class="user-profile">
   <div class="sidebar-username"><a class="leaner-profile-link" href="${learner_profile_page_url}">${django_user.username | h}</a></div>
   <div class="sidebar-user-roles">
-      ${_(', ').join(sorted(set(map(_, [role.name for role in django_user.roles.all()]))))}
+    <% role_names = django_user.roles.order_by("name").values_list("name", flat=True).distinct() %>
+    ${", ".join(_(role_name) for role_name in role_names)}
```

Comment:
This change looks good to me. I have a couple of concerns:
1. I don't know how well this code is covered, so I can't be sure that this works as is. Either we should find a test that does cover it, or we'll need to write a new one. Have you verified it locally?
2. This was already an issue, but I don't think the way we drop in the comma is going to localize cleanly.

I know that @explorerleslie had some plans to clean up this page now that we have learner profiles, so maybe #2 can get cleaned up later.